src/pygats/recog.py

```diff
@@ -0,0 +1,360 @@
+"""
+module with data classes.
+"""
+
+from dataclasses import dataclass
+import pytesseract
+import re
+import pyperclip
+import pytesseract
+from Levenshtein import ratio
+
+
+@dataclass
+class SearchedText:
+    """
+    Data class to store text with lang to be passed as parameters
+    for Tesseract function
+    """
+    text: str
+    lang: str
+
+
+def find_cropped_text(img, txt, skip=0, one_word = False):
+    """
+    Find text in image. Several passes are used.
+    First time found area with text on image and then
+    every area passed through recognition again to improve recognition results
+
+    Args:
+        img (PIL.Image): image to search text in
+        txt (pygats.recog.SearchedText): text to search
+        skip (int, optional): number of occurrences of the text to skip.
+
+    Returns:
+        (left, top, width, height, found):
+            left (int): left coordinate of the text bounding box
+            top (int): top coordinate of the text bounding box
+            width (int): width of the text bounding box
+            height (int): height of the text bounding box
+            found (bool): whether the text is found in the image
+
+    """
+    recognized = pytesseract.image_to_data(img, txt.lang).split('\n')
+    if one_word is False:
+        combine_words_in_lines(recognized)
+    ret_tuple = (-1, -1, -1, -1, False)
+    for line in recognized[1:]:
+        splitted = line.split('\t')
+        if len(splitted) == 12 and splitted[11].find(txt.text) != -1:
+            print(f'Найден текст {splitted[11]}')
+            ret_tuple = (int(splitted[6]), int(splitted[7]), int(
+                splitted[8]), int(splitted[9]), True)
```

Comment:
must either put the lower arguments under the arguments of the upper line, or move the arguments of the upper line to the next line